apps/passport-client/pages/index.tsx

```diff
@@ -115,24 +115,81 @@ class App extends React.Component<object, AppState> {
     } as Partial<AppState>;
   }
 
+  BG_POLL_INTERVAL_MS = 1000 * 60;
+  lastBackgroundPoll = 0;
+  activePollTimout: NodeJS.Timeout | undefined = undefined;
+
```

Comment:
prefer to declare these fields at the top with all the other declarations; would like access modifiers + a `readonly` modifier on the `BG_POLL_INTERVAL_MS` field